.travis.yml

```diff
@@ -4,27 +4,29 @@ node_js:
   - "0.10"
   - "0.12"
   - "4"
+  - "6"
+  - "7"
 
-matrix:
-  include:
-    - node_js: "6"
-      addons:
-        firefox: "49.0"
-      env: BROWSER=true MAKE_TEST=true
+#matrix:
+#  include:
+#    - node_js: "6"
+#      addons:
+#        firefox: "49.0"
+#      env: BROWSER=true MAKE_TEST=true
```

Comment:
I disabled this thinking it was an issue with karma, but it's a syntax error.  I'll investigate further and re-enable.  We could test both 45 (ESR, no `async` support) and 52 (`async` supported).